tests/analysis/chemenv/coordination_environments/test_coordination_geometries.py

```diff
@@ -89,7 +89,7 @@ def test_coordination_geometry(self):
         assert cg_oct.IUCr_symbol_str == "[6o]"
 
         cg_oct.permutations_safe_override = True
-        assert cg_oct.number_of_permutations == 720.0
```

Comment:
this should be int?

Reply:
Yes.

Reply:
😆 Thanks for chiming in, I was just going through all changes again